Modules/CTP/basic-ctp.json

```diff
@@ -0,0 +1,76 @@
+{
+  "qc": {
+    "config": {
+      "database": {
+        "implementation": "CCDB",
+        "host": "ccdb-test.cern.ch:8080",
+        "username": "not_applicable",
+        "password": "not_applicable",
+        "name": "not_applicable"
+      },
+      "Activity": {
+        "number": "42",
+        "type": "2"
+      },
+      "monitoring": {
+        "url": "infologger:///debug?qc"
+      },
+      "consul": {
+        "url": "http://consul-test.cern.ch:8500"
+      },
+      "conditionDB": {
+        "url": "ccdb-test.cern.ch:8080"
+      }
+    },
+    "tasks": {
+      "MyRawDataQcTask": {
+        "active": "true",
+        "className": "o2::quality_control_modules::ctp::RawDataQcTask",
+        "moduleName": "QcCTP",
+        "detectorName": "CTP",
+        "cycleDurationSeconds": "10",
+        "maxNumberCycles": "-1",
+        "": "The other type of dataSource is \"direct\", see basic-no-sampling.json.",
+        "dataSource": {
+          "type": "dataSamplingPolicy",
+          "name": "tst-raw"
+        },
+        "taskParameters": {
+          "myOwnKey": "myOwnValue"
+        },
+        "location": "remote",
+        "saveObjectsToFile": "test.root",      "": "~/alice/QualityControl/Modules/CTP/"
+      }
+    },
+    "checks": {
+      "RawDataQcCheck": {
+        "active": "true",
+        "className": "o2::quality_control_modules::ctp::RawDataQcCheck",
+        "moduleName": "QcCTP",
+        "policy": "OnAny",
+        "detectorName": "CTP",
+        "dataSource": [{
+          "type": "Task",
+          "name": "MyRawDataQcTask",
+          "MOs": ["example2"]
+        }]
+      }
+    }
+  },
+  "dataSamplingPolicies": [
+    {
+      "id": "tst-raw",
+      "active": "true",
+      "machines": [],
+      "query": "random:TST/RAWDATA/0",
```

Comment:
Are you sending your data under `TST/RAWDATA` ? shouldn't it be under something like `CTP/RAWDATA` ?

Reply:
I don't sample data (yet) so I left it as it was from example. I am changing tst to ctp in both cases.

Reply:
but if you are not sampling, what data are you monitoring ?

Reply:
This version is reading raw files locally - i.e. raw file is created from simulation (then digits, then raw file). I wanted to have it while waiting for interface by Sylvain. The payload structure will be the same in the real data so it is not waste of time. If I correctly understand, the data sampling is needed when you want monitor only fraction of data? here I monitor all (they are from simulation).